### DISCUSSION TITLE
Freshly installed UVR won't start  :(

### DISCUSSION THREAD

**Discussion by MarkGuit** [Anjok07/ultimatevocalremovergui#1411] [Q&A] [answered]
As my title says, I just installed UVR5 on my WIndows 10 system but it's giving me this error each time I try to start the program:

Traceback (most recent call last):
  File "UVR.py", line 180, in load_data
FileNotFoundError: [Errno 2] No such file or directory: 'data.pkl'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "UVR.py", line 288, in <module>
  File "UVR.py", line 187, in load_data
  File "UVR.py", line 169, in save_data
PermissionError: [Errno 13] Permission denied: 'data.pkl'

Any help in getting the program to work would be appreciated!

**Comment by MarkGuit** [comment, marked as answer]
I tried to install it on a second computer, but this time I had a hunch.
Instead to giving UVR5 a different folder to install to, I used the one it suggested.

It installed properly and opened properly!
Go figure.  :)

So I went back to my main computer, uninstalled it, and installed it again.
But this time I installed it to the folder it suggested.

It installed properly and opened properly!
I haven't done anything else with the program yet, but at least now I have
something to work with.  :)

**Comment by wayne931121** [comment]
Hello, I meet the same problem, I found solution that that program I install in C:\Program Files\ need administrator to run to have the permission to use data.pkl or it will permission denied.
